src/modules/inhibit/config.rs

```diff
@@ -0,0 +1,126 @@
+use serde::{Deserialize, Deserializer};
+use std::time::Duration;
+
+/// Command to control inhibit state.
+///
+/// **Valid options**: `toggle`, `cycle`
+#[derive(Debug, Clone, Copy, Deserialize)]
+#[cfg_attr(feature = "extras", derive(schemars::JsonSchema))]
+#[serde(rename_all = "lowercase")]
+pub enum InhibitCommand {
+    /// Toggle inhibit on/off
+    Toggle,
+    /// Cycle to next duration
+    Cycle,
+}
+
+#[derive(Debug, Deserialize, Clone)]
+#[cfg_attr(feature = "extras", derive(schemars::JsonSchema))]
+#[serde(default)]
+pub struct InhibitModule {
+    /// List of durations to cycle through.
+    /// Use `0` for infinite inhibit.
+    /// Format: `HH:MM:SS` (e.g., `01:30:00` for 1 hour 30 minutes)
+    ///
+    /// **Default**: `["00:30:00", "01:00:00", "01:30:00", "02:00:00", "0"]`
+    #[serde(deserialize_with = "deserialize_durations")]
+    #[cfg_attr(feature = "extras", schemars(with = "Vec<String>"))]
+    pub(super) durations: Vec<Duration>,
+
+    /// The default duration to use when starting inhibit.
+    /// Must be one of the values in `durations`.
+    /// Format: `HH:MM:SS` (e.g., `02:00:00` for 2 hours)
+    ///
+    /// **Default**: `"02:00:00"`
+    #[serde(deserialize_with = "deserialize_duration")]
+    #[cfg_attr(feature = "extras", schemars(with = "String"))]
+    pub(super) default_duration: Duration,
+
+    /// Command to execute on left click.
+    ///
+    /// **Valid options**: `toggle`, `cycle`
+    /// <br>
+    /// **Default**: `toggle`
+    pub(super) on_click_left: Option<InhibitCommand>,
+
+    /// Command to execute on right click.
+    ///
+    /// **Valid options**: `toggle`, `cycle`
+    /// <br>
+    /// **Default**: `cycle`
+    pub(super) on_click_right: Option<InhibitCommand>,
+
+    /// Command to execute on middle click.
+    ///
+    /// **Valid options**: `toggle`, `cycle`
+    /// <br>
+    /// **Default**: `null`
+    pub(super) on_click_middle: Option<InhibitCommand>,
+
+    /// Format string when inhibit is active.
+    /// `{duration}` token shows remaining/selected time.
+    ///
+    /// **Default**: `"☕ {duration}"`
+    pub(super) format_on: String,
+
+    /// Format string when inhibit is inactive.
+    /// `{duration}` token shows selected duration.
+    ///
+    /// **Default**: `"💤 {duration}"`
+    pub(super) format_off: String,
+
+    /// See [layout options](module-level-options#layout).
+    #[serde(flatten)]
+    pub(super) layout: LayoutConfig,
+
+    /// See [common options](module-level-options#common-options).
+    #[serde(flatten)]
+    pub common: Option<CommonConfig>,
+}
+
+impl Default for InhibitModule {
+    fn default() -> Self {
+        Self {
+            durations: ["00:30:00", "01:00:00", "01:30:00", "02:00:00", "0"]
+                .into_iter()
+                .map(|s| parse_duration(s).expect("default durations entry invalid"))
+                .collect(),
+            default_duration: parse_duration("02:00:00").expect("default duration invalid"),
```

Comment:
I think the default durations are fine, but a shorter default *default* might be preferable


```suggestion
            default_duration: parse_duration("00:30:00").expect("default duration should be valid"),
```